doc/engine/Running-the-OpenQuake-Engine-on-Multiple-Nodes.md

```diff
@@ -160,9 +145,9 @@ If using `supervisord` (or similar) to execute `celeryd` at boot time please ens
 Storage requirements depend a lot on the type of calculations run. On a worker node you will need just the space for the operating system, the logs and the OpenQuake installation: less than 20GB are usually enough. Workers can be also diskless (using iSCSI or NFS for example).
 
 On the master node you will also need space for:
+- the users' **home** directory (usually located under `/home`). The users home folders will contains the calculations exported results and the calculations datastore (`hdf5` files located in the `oqdata` folder)
 - the PostgreSQL DB (on Ubuntu usually located under `/var/lib/postgres`)
```

Comment:
Actually the exported results will in the directory `export_dir` which can be everywhere. They go in the home only if `export_dir` is not specified in the job.ini file.